docs/components/avatar/avatar.css

```diff
@@ -0,0 +1,48 @@
+#avatar-container {
```

Comment:
container used for example in documentation so need not be added in component css but in a seperate CSS file for docuemtation site.

Reply:
+1